clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinatorTest.java

```diff
@@ -1220,6 +1226,180 @@ public void testProtocolMetadataOrder() {
         }
     }
 
+    @Test
+    public void testCloseAutoAssignment() throws Exception {
+        ConsumerCoordinator coordinator = prepareCoordinatorForCloseTest(true, true);
+        gracefulCloseTest(coordinator);
+    }
+
+    @Test
+    public void testCloseManualAssignment() throws Exception {
+        ConsumerCoordinator coordinator = prepareCoordinatorForCloseTest(true, true);
```

Comment:
I can't make out the difference between this test and the previous one. looks identical from the code.